cmd/spartakus/volunteer.go

```diff
@@ -42,6 +43,7 @@ func (_ volunteerSubProgram) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&volunteerConfig.database, "database",
 		"https://spartakus.k8s.io", "Send reports to this database; use --print-databases for a list of options")
 	fs.BoolVar(&volunteerConfig.printDatabases, "print-databases", false, "Print database options and exit")
+	fs.StringVar(&volunteerConfig.extensionsPath, "extensions", "", "Path to a file of additional metrics to report; leave unset to report no additional metrics")
```

Comment:
Hmm you bring up two good points.
1. dealing with race condition between extension writers and spartakus
2. mitigating race conditions between multiple extensions writers

The temp file + rename solution solves the race condition for a single writer + spartakus but still allows multiple writers to overwrite eachother. Regarding 2: I had not considered this use case. I spoke with @sym3tri and we propose specifying an extensions directory instead of a path to a file. Spartakus will attempt to parse extensions from all of the files in that dir and merge them best effort. This allows different writers to write to separate files and not step on each other. Thoughts?